runtime/proto/corfu_store_metadata.proto

```diff
@@ -19,13 +19,18 @@ message TableName {
     string table_name = 2;
 }
 
-// Descriptor to provide self describing schemas for the records.
+// Describe the protos in a file.
 message TableDescriptors {
     // File Descriptor protobufs map.
     // Contains file descriptors of key, value, metadata and their dependencies.
     map<string, google.protobuf.FileDescriptorProto> fileDescriptors = 1;
 }
 
+message KeyValueDefaultRecord {
+    google.protobuf.Any key = 1;
+    google.protobuf.Any value = 2;
```

Comment:
What about metadata? the vast majority of our users deploy with Key, Value and Metadata right?
Also if you need this information can we just add these fields into the TableDescriptors message above as additional fields. Doing so will eliminate the need to have a separate corfu table, it will also reduce the memory footprint since we only have to cache the table names twice (TableRegistry and ProtobufDescriptorTable)?

Reply:
done